src/Components/CommentCard.jsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+const CommentCard = ({ comment }) => {
+  return (
+    <div>
+      <p><strong>{comment.author}</strong> says:</p>
+      <p>{comment.body}</p>
+      <p>votes: {comment.votes}</p>
+      <p>{comment.created_at.slice(0,10)}</p>
```

Comment:
Consider using the time here too, you may have to format "created_at" further to display it.